cmd/image-loader/operator.go

```diff
@@ -38,7 +38,7 @@ func loadKubermaticConfiguration(log *zap.SugaredLogger, filename string) (*kube
 	}
 
 	config := &kubermaticv1.KubermaticConfiguration{}
-	if err := yaml.Unmarshal(content, &config); err != nil {
+	if err := yaml.UnmarshalStrict(content, &config); err != nil {
```

Comment:
I have a question about this, from what I understand, this(and others like this) will fail if there is a field that does not fit the struct. 

So what in the case when somebody wants to add a new field to the CRD like KubermaticConfiguration, and use it in for example in seed-controller-manager. To test this on dev, the change has to be present in all the components, otherwise they will be throwing errors?

Reply:
Good point. This is not forward compatible.

Reply:
ahh good point, we have openapi CRDs so its the same